### PR TITLE
PE parser: add unit tests and remove usage of arrow module

### PR DESCRIPTION
**Pull request by gianantoniopini**
## Issue

#6135 

## Description

This PR adds unit tests for the PE parser and removes usage of the `arrow` module from the PE parser.

### Double check

- [x] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: parsers/test/test_PE.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        )
+
+        self.assertMatchSnapshot(
+            [
+                {
+                    "datetime": element["datetime"].isoformat(),
+                    "production": element["production"],
+                    "source": element["source"],
+                    "zoneKey": element["zoneKey"],
+                    "sourceType": "measured",
```
Comment (VIKTORVAV99) [inline parsers/test/test_PE.py:41]:
```suggestion
                    "sourceType": element["sourceType"].value,
```
This just brings it in line with our other tests and makes it easy to update if we ever change this.

**Review by VIKTORVAV99 — COMMENTED** [review]
Looks good!

Just one tiny comment to take a look at then I'll merge this.

File: parsers/test/mocks/PE/response_20240205.json
Comment (VIKTORVAV99) [inline parsers/test/mocks/PE/response_20240205.json:1]: One thing we can do if we want to reduce the size of these are minifying the files, it should have no impact on the tests.

**Comment by VIKTORVAV99** [comment]
Merging this as is right now as we have a follow up PR that converts it to use parser event lists as well.

**Review by VIKTORVAV99 — APPROVED** [review]

**Comment by gianantoniopini** [comment]
👍

**Pull request merged**